BedrockServer.h

```diff
@@ -91,6 +91,49 @@ class BedrockServer : public SQLiteServer {
     // can do about these, except not shut down. If we're standing down, we could keep them, but they break our
     // checking against whether the main command queue is empty, and fall into the same category of us being unable to
     // distinguish them from commands that may have already started.
+    //
+    // Notes on timing out a shutdown.
+    // Here's how timing out a shutdown works:
+    // 1. When beginShutdown() is called, it sets a one minute timer. It proceeds to shut down the node as normal.
```

Comment:
Isn't this `_beginShutdown()`? We have both, `beginShutdown()` for SQLiteNode and `_beginShutdown()` for BedrockServer, and if I understand correctly we shut down BedrockServer, then set the timeout for SQLiteNode to whatever is left (2. below).